mysql-plugin/widgets/Mysql-batchsource.json

```diff
@@ -107,9 +107,80 @@
         }
       ]
     },
+    {
+      "label": "SSL",
+      "properties": [
+        {
+          "label": "Use SSL",
+          "name": "useSSL",
+          "widget-type": "select",
+          "widget-attributes": {
+            "default": "If available",
+            "values": ["No", "If available", "Require"]
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Keystore URL",
+          "name": "clientCertificateKeyStoreUrl"
+        },
+        {
+          "widget-type": "password",
+          "label": "Keystore password",
+          "name": "clientCertificateKeyStorePassword"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Truststore URL",
+          "name": "trustCertificateKeyStoreUrl"
+        },
+        {
+          "widget-type": "password",
+          "label": "Truststore password",
+          "name": "trustCertificateKeyStorePassword"
+        }
+      ]
+    },
     {
       "label": "Advanced",
       "properties": [
+        {
+          "widget-type": "toggle",
+          "label": "Use compression protocol",
+          "name": "useCompression",
+          "widget-attributes": {
+            "on": {
+              "value": "true",
+              "label": "On"
```

Comment:
If this is labeled 'Use Compression', then 'Yes' and 'No' are better labels (and more consistent with 'Use SSL' above). If it were labeled 'Compression', then 'On' and 'Off' make more sense. For consistency, let make it 'Use Compression' with 'Yes' and 'No'.